ui/src/app/views/concept-homepage/component.ts

```diff
@@ -144,4 +183,33 @@ export class ConceptHomepageComponent implements OnInit {
   get searchTermNotLongEnough() {
     return this.searchTerm === undefined || this.searchTerm.length < 3;
   }
+
+
+
+  filterList() {
+    if (this.blockMultipleSearchFromFilter) {
+      this.blockMultipleSearchFromFilter = false;
+      return;
+    }
+    this.blockMultipleSearchFromFilter = true;
+    let standardConceptFilter: StandardConceptFilter;
+    if (this.standardConceptsOnly) {
+      standardConceptFilter = StandardConceptFilter.STANDARDCONCEPTS;
+    } else {
+      standardConceptFilter = StandardConceptFilter.ALLCONCEPTS;
+    }
+    this.searchLoading = true;
+    const request = {
+      query: this.currentSearchString,
+      standardConceptFilter: standardConceptFilter,
+      domain: this.selectedDomain,
+      vocabularyIds: this.vocabularies
+        .filter(vocabulary => vocabulary.selected).map(vocabulary => vocabulary.vocabularyId),
+    };
+    this.conceptsService.searchConcepts(this.wsNamespace, this.wsId, request)
```

Comment:
Error callback?

Reply:
Haven't implemented errors here yet, but I can do that.